pages/docs/environments/local-development.mdx

```diff
@@ -1,7 +1,53 @@
 import DocsLayout from "@components/layouts/DocsLayout.js";
 import Link from "@components/Link";
 
-## Quickstart for local development
+<div id="docker-compose">
```

Comment:
I think you can use the headings for deep linking (`##`)

Reply:
`<div id="docker-compose">` this I'm using for `https://linkfree.io/docs/environments/local-development#docker-compose` in the readme file. Having an id for this section will allow us to auto scroll to the docker-compose section when user clicks on the link.

Reply:
I think if we update the mdx component for the headings, then we won't need the extra html and we can use this in other places also https://github.com/EddieHubCommunity/LinkFree/tree/main/components/mdx

Reply:
I have added a new component named Heading and used it. Have to say it is a far better approach than what I was using.

Reply:
This is how it looks now: 


![image](https://user-images.githubusercontent.com/36790357/233436984-105610e9-ea55-4b41-bb3c-966597c0d1cf.png)


![image](https://user-images.githubusercontent.com/36790357/233436969-eb78625b-8bc1-4c68-896e-b04183041bb7.png)

Reply:
Hello! There is no need to create a header component for linking from external URL or within documents as markdown supports this natively.

Requesting removal of the header component and staying with pure markdown to remain consistent throughout the project and decrease complexity by adding more code.

Here is an example of how to adjust your headings in pure markdown to allow for linking.  Note that if you have a 2 word header, the url should utilize the first word in the header for the link

[markdown with linking example](https://github.com/fefong/markdown_readme/blob/master/README.md?plain=1)

example header with url for 2 word scenario:  

LOCAL DEVELOPMENT

would be url.com/page-name#local

If you would like assistance making these changes, let me know and I"m happy to help further

Reply:
Hey Amanda and Eddie, I have gone through all the reviews and I'm making appropriate changes.
I will be pushing my changes in some time.